tests/acceptance/features/coreApiVersions/fileVersions.feature

```diff
@@ -295,7 +295,7 @@ Feature: dav-versions
     And user "Brian" has accepted share "/sharefile.txt" offered by user "Alice"
     When user "Brian" uploads file with content "Second content" to "/Shares/sharefile.txt" using the WebDAV API
     Then the HTTP status code should be "204"
-    And the version folder of file "/Shares/sharefile.txt" for user "Brian" should contain "1" element
+    And the version folder of file "/Shares/sharefile.txt" for user "Brian" should contain "-1" element
     And the version folder of file "/sharefile.txt" for user "Alice" should contain "1" element
```

Comment:
I would change it as "2"
and change code here https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/bootstrap/FilesVersionsContext.php#L208

If you don't mind I can take care of this PR.